.github/workflows/blank.yml

```diff
@@ -12,3 +12,14 @@ jobs:
       with:
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
         configuration-path: "./issue-rules.yml"
+
+    - name: Set Node.js 10.x
+      uses: actions/setup-node@master
+      with:
+        version: 10.x
+
+    - name: File bugs
+      run: node ci/file-bugs.js
+      env:
+        GITHUB_PAT: "${{ secrets.GITHUB_TOKEN }}"
+        AZP_PAT: "${{ secrets.AZP_TOKEN }}"
```

Comment:
I added this secret, its just my PAT for now. That'll expire in a year though, any thoughts on a way to do this more sustainably? It may be that there just isn't a way.

Reply:
I don't know a ton about how our OAuth stuff works, but you might be able to register as an app?

Reply:
From https://docs.microsoft.com/en-us/azure/devops/integrate/get-started/authentication/oauth?view=azure-devops I think I'd still have to refresh the token unfortunately. I guess we try to avoid giving something infinite access to resources, which I guess makes sense.